src/components/TodoItem.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/destructuring-assignment */
+import React, { useState, useEffect } from 'react';
```

Comment:
- [x] Linters are tools that perform static analysis on software to detect and report adherence to a set of coding best practices. Instead of disabling eslint, it would be preferable to resolve the issue.